static/docs/get-started/pipeline.md

```diff
@@ -30,12 +30,14 @@ Let's commit DVC-files that describe our pipeline so far:
 
 ```dvc
 $ git add data/.gitignore .gitignore featurize.dvc train.dvc
-$ git commit -m "add featurization and train steps to the pipeline"
```

Comment:
same - it's a specific pipeline in this case

Reply:
Will revert.